package-lock.json

```diff
@@ -5914,7 +5923,7 @@
     "zip-stream": {
       "version": "2.0.1",
       "resolved": "https://registry.npmjs.org/zip-stream/-/zip-stream-2.0.1.tgz",
-      "integrity": "sha512-c+eUhhkDpaK87G/py74wvWLtz2kzMPNCCkUApkun50ssE0oQliIQzWpTnwjB+MTKVIf2tGzIgHyqW/Y+W77ecQ==",
+      "integrity": "sha1-SKBiSIr+kd2kL4I3APrliXU8zTQ=",
```

Comment:
I don't think sha1 is okay. It looks like a usual npm lock mess. Could you try to
```cmd
rm -rf node_modules
npm cache clean --force
git checkout origin/master -- package-lock.json
npm install ...
```